src/LiveComponent/doc/index.rst

```diff
@@ -2453,6 +2453,40 @@ The parent-child system is *smart*. And with a few tricks
 (:ref:`such as the key prop for lists of embedded components <rendering-quirks-with-list-of-embedded-components>`),
 you can make it behave exactly like you need.
 
+.. _passing-blocks:
+
+Passing Blocks to Components
```

Comment:
I think this is in the wrong section - it's right in the middle of the docs on "nested components" - i.e. parent-child setup. 

Also, WDYT about:

`Passing Content (Blocks) to Components`

From a user's perspective, I am "passing content into my component". Blocks are the mechanism, but not the star. We should update the same title in Twig components.

Reply:
Oh yeah, you're right 🙈 
I just put it right after nested blocks, because that's also where it is in the Twig Component docs, but indeed in the Live Components there are subsections. I placed it behind those subsections now.

Reply:
I also changed the section titles.